packages/html/src/chart-wizard/chart-wizard-splitter.tsx

```diff
@@ -0,0 +1,39 @@
+import { classNames } from '../misc';
+import { Splitter, KendoSplitterProps } from '../splitter';
+
+export const CHARTWIZARDSPLITTER_CLASSNAME = `k-chart-wizard-splitter`;
+
+const states = [];
+
+const options = {};
+
+const defaultProps = {};
+
+export const ChartWizardSplitter = (
+    props: KendoSplitterProps &
+        React.HTMLAttributes<HTMLDivElement>
+) => {
+    const {
+        children,
+        ...other
+    } = props;
+
+    return (
+        <Splitter style={{ display: "flex", height: "100%", width: "100%", overflow: "hidden" }}
```

Comment:
Could we scope these styles through `k-chart-wizard-splitter` instead of adding them inline?